app/LearnedWord.php

```diff
@@ -12,4 +13,17 @@ public function lesson()
     {
     	return $this->belongsTo(Lesson::class);
     }
+
+    public function isCorrect()
+    {
+    	$returnValue = false;
+
+    	$item = Item::find($this->item_id);
```

Comment:
Instead of instantiating an Item Object Model in LearnedWord model you can declare the relationship that LearnedWord belongsTo Item and from there you can access the item object. similar to this one:
[![Screenshot from Gyazo](https://gyazo.com/f2cbe61acd79d2be0703f7a3c7d74170/raw)](https://gyazo.com/f2cbe61acd79d2be0703f7a3c7d74170)